Doc/c-api/typeobj.rst

```diff
@@ -163,9 +163,9 @@ Quick Reference
 
 .. [#cols] Columns:
 
-   **"O"**:  set on :c:type:`PyBaseObject_Type`
+   **"O"**:  set on :c:data:`PyBaseObject_Type`
```

Comment:
```:c:data:`PyBaseObject_Type` ``` is also not found. Can it be documented or ab alternative used?

Reply:
Yes, it is not currently documented. But this out of scope of this PR which simply fixes markup. When it be documented, all these reference will instantly became links.